### ISSUE TITLE
Unknown value 'OUT' in HdmiOut

### ISSUE THREAD

**Issue by rootnegativ1** [mvdwetering/yamaha_ynca#297]
**Describe the bug**
I get a warning each time Home Assistant restarts or the receiver (RX-V671) turns on

**Diagnostics**
[config_entry-yamaha_ynca-f7cd6b8a90dbc211823d68bedee7ff3e.json](https://github.com/user-attachments/files/16042334/config_entry-yamaha_ynca-f7cd6b8a90dbc211823d68bedee7ff3e.json)

**Logging**
```
Logger: ynca.enums
Source: /usr/local/lib/python3.12/site-packages/ynca/enums.py:117
First occurred: 3:17:19 PM (1 occurrences)
Last logged: 3:17:19 PM

Unknown value 'OUT' in HdmiOut
```

**Environment**
- Home Assistant Core version:  2024.6.3
- Home Assistant operating environment (OS/Container/Supervised/Core):  Supervised
- Integration version: 7.9.0
- Last working version (if known): unknown

**Comment by mvdwetering** [comment]
The command it is giving the warning for I have only seen before to switch HDMI outputs for receivers with 2 HDMI OUT.

Since RX-V671 seems to have 1 HDMI output I am wondering what this command is being used for. I browsed a bit through the [manual](https://usa.yamaha.com/files/download/other_assets/6/321916/RX-V671_Manual.pdf), but it did not become clear to me. Only mention I found was about Audio output on HDMI out on P95, but that seems unrelated (also because options are Off and On, not OUT)

Is it possible to toggle HDMI output On/Off or something?
If that is the case could you do that and send the diagnostics output again so I can see what values the receiver sends when toggling such a setting.?

**Comment by mvdwetering** [comment]
This is fixed in release v7.10.0 of the integration which is just released.
